playbooks/aws/openshift-cluster/launch.yml

```diff
@@ -17,20 +17,37 @@
       instances: "{{ etcd_names }}"
       cluster: "{{ cluster_id }}"
       type: "{{ k8s_type }}"
+      g_sub_host_type: "{{ k8s_type }}"
```

Comment:
I'd prefer just setting this to "default" for non-node host-types.

So this (untested):
```
g_sub_host_type: default
```